changelog.txt

```diff
@@ -1,3 +1,295 @@
+Feature: Sandbox option to lock station ratings at 100% (#11346)
+Feature: Setting to use real-time "wallclock" as timekeeping units (#11341)
+Feature: Setting to automatically restart server based on hours played (#11142)
+Feature: Add config option to set default company secondary colour for new games (#11068)
+Feature: Transparency option for cost and income indicators (#11001)
+Feature: Create group of vehicles from manage vehicle list button (#10890)
+Feature: Show coverage highlight the same as stations when adding waypoints (#10875)
+Feature: Show the number of industries already built in the Fund New Industry window (#10806)
+Feature: Add search filter and name text to build waypoint window (#10786)
+Feature: Setting to disallow level crossings with competitors (#10755)
+Feature: Opt-in survey when leaving a game (#10719)
+Feature: Replace buying/selling company shares with hostile takeovers of AI companies (#10709, #10914)
+Feature: Settings to scale cargo production of towns and industries (#10606)
+Feature: Separate rail/road and sea/air velocity units, and add knots (#10594)
+Feature: Region-based pathfinder for ships (#10543)
+Feature: Filter engine build menu by name and NewGRF extra text (#10519)
+Feature: Industry directory text filter (#10518)
+Feature: Ctrl+Click to reset late counter for the entire vehicle group (#10464)
+Feature: Orientation of rail and road depots can be changed (#9642)
+Feature: Display help and manuals in-game (#7786)
+Feature: [NewGRF] Town production effect and multiplier (#11947)
+Feature: [NewGRF] Randomize direction of rail vehicle on build based on probability callback (#11489)
+Feature: [NewGRF] Related Act2 objects for airports and airport tiles (#11282)
+Feature: [NewGRF] Allow higher max speeds for ships (#10734)
+Feature: [NewGRF] Increase limit of objects/stations/roadstops per NewGRF (#10672)
+Feature: [NewGRF] Road stops (#10144)
+Feature: [Script] Goal destination can be updated (#10817)
+Add: Argument for console command "restart" to use either current or newgame settings (#11962, #11963)
+Add: {CURRENCY_SHORT} only did k / m suffix. Add bn / tn and make translatable (#11921)
+Add: Show in multiplayer the amount of hours a game has been unpaused (#11886)
+Add: Allow loading heightmaps from command-line (#11870)
+Add: List_[scenario|heightmap] and load_[scenario|height] console commands (#11867)
+Add: Latvian Lats currency (#11691)
+Add: Horizontal scroll for script debug log (#11597)
+Add: GUI options to select sprite font and AA mode for all fonts (#11593)
+Add: Website button for basesets in Game Options window, the Game Script settings window and AI settings window (#11512)
+Add: Debug tool to draw widget outlines (#11401)
+Add: [Emscripten] Support for bootstrapping (#11109)
+Add: Hotkey to focus town / industry directory filter box (#11030)
+Add: {COLOUR} control code to allow colour changes at runtime (#10687)
```

Comment:
If NewGRF and/or GS can use this, it could be moved to their section. If not and it's only internal, I don't think it belongs in the changelog.